src/Apps/Artwork/Components/ArtworkSidebar2/ArtworkSidebar2.tsx

```diff
@@ -74,7 +75,9 @@ export const ArtworkSidebar2: React.FC<ArtworkSidebarProps> = props => {
 
       <Spacer mt={2} />
       <ArtworkSidebar2DetailsFragmentContainer artwork={artwork} />
-      <Spacer mt={2} />
+      <Separator />
```

Comment:
**question**: won't we have a problem with the fact that `Spacer` will be rendered twice ([here](https://github.com/artsy/force/blob/a7e04afef41e8cee76dc9fd09e640bcaa364b643/src/Apps/Artwork/Components/ArtworkSidebar2/ArtworkSidebar2.tsx#L78) and [here](https://github.com/artsy/force/blob/a7e04afef41e8cee76dc9fd09e640bcaa364b643/src/Apps/Artwork/Components/ArtworkSidebar2/ArtworkSidebar2.tsx#L95)) if there is nothing to render in edition sets and `artworkEcommerceAvailable` is `false`?

Reply:
I will open a follow up pr after all the tickets are addressed fixing all these 🤞